src/main/java/core/basesyntax/ColorEnum.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum ColorEnum {
```

Comment:
we don't add enum to naming
```suggestion
public enum Color {
```